.github/workflows/staticcheck.yml

```diff
@@ -21,6 +21,9 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v3
 
+    - name: Run gofmt
+      run: "gofmt -d $(find . -maxdepth 1 -type d -and -not -name '.git' -and -not -name '.' -or -name '*.go')"
```

Comment:
Do we use `gofmt` or `goimports`?

Reply:
I'd be happy with only gofmt for now.

You're asking whether we want to use `goimports --format-only` instead here? Could.

Reply:
I mean I use `goimports` locally, if we want to not confuse new people joining the team I would like tools to be consistent across CI and local dev. So yes `goimports --format-only`.

Reply:
done